.travis.yml

```diff
@@ -12,6 +12,7 @@ notifications:
     template:
       - "<!here>"
       - "%{repository_slug} (%{build_number}) : %{message}"
+      - "Commit: %{commit_subject} by %{author}"
       - "Build details: %{build_url}"
```

Comment:
Since this is a cron job, you might want to say "last commit" since it doesn't have to be the one causing the failure.

Reply:
Good point! Updated